website/blog/2021-06-28-release-0-2.md

```diff
@@ -0,0 +1,43 @@
+---
+slug: release-sb
```

Comment:
this is id in link `/blog/<slug>`, change to "release-sb-v0.2"?

Reply:
sure. changed.